src/components/UnstyledLink/UnstyledLink.tsx

```diff
@@ -20,11 +20,18 @@ export const UnstyledLink = memo(
       return <LinkComponent {...unstyled.props} {...props} />;
     }
 
-    const {external, url, ...rest} = props;
+    const {external, url, accessibilityLabel, ...rest} = props;
     const target = external ? '_blank' : undefined;
     const rel = external ? 'noopener noreferrer' : undefined;
     return (
-      <a target={target} {...rest} href={url} rel={rel} {...unstyled.props} />
+      <a
```

Comment:
Per other comment, Let's keep with using `aria-label` as the prop here, which I believe means we don't need any of this - as it'll already be covered by the `{...rest}`